<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpatialDecisionDockWidgetBase</class>
 <widget class="QDockWidget" name="SpatialDecisionDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDSS Template</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="sampleWidgets">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="dataTab">
       <attribute name="title">
        <string>Events</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="openScenarioButton">
            <property name="text">
             <string>Open Scenario</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="saveScenarioButton">
            <property name="text">
             <string>Save Scenario</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="selectLayerLabel">
            <property name="text">
             <string>Select layer:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="selectAttributeCombo"/>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="selectLayerCombo"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="selectAttributeLabel">
            <property name="text">
             <string>Select attribute:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="showButton">
            <property name="text">
             <string>Get Values:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QListWidget" name="valueWidget"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="analysisTab">
       <attribute name="title">
        <string>Units</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="1" column="0">
         <widget class="QPushButton" name="shortestRouteButton">
          <property name="text">
           <string>Shortest Route</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="clearRouteButton">
          <property name="text">
           <string>Clear Routes</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="serviceAreaButton">
          <property name="text">
           <string>ServiceArea</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="serviceAreaCutoffEdit">
          <property name="text">
           <string>1500</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="bufferButton">
          <property name="text">
           <string>Buffer</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="bufferCutoffEdit">
          <property name="text">
           <string>800</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="makeIntersectionButton">
          <property name="text">
           <string>Intersection</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="selectBufferButton">
          <property name="text">
           <string>Select in Buffer</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="selectRangeButton">
          <property name="text">
           <string>Select in Range</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QLineEdit" name="expressionEdit">
          <property name="text">
           <string>id &lt; 3</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="expressionSelectButton">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="expressionFilterButton">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="assignButton">
          <property name="text">
           <string>Assign</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="clearAssignmentButton">
          <property name="text">
           <string>ClearAssignment</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="windTab">
       <attribute name="title">
        <string>Wind conditions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="chartLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
